block/cardano-sl-block.cabal

```diff
@@ -157,22 +157,22 @@ library
   build-tools: cpphs >= 1.19
   ghc-options: -pgmP cpphs -optP --cpp
 
-Benchmark bench-block
-    type: exitcode-stdio-1.0
-    main-is: Block.hs
-    hs-source-dirs: bench
-    ghc-options: -O2
-    build-depends:   base
-                   , bytestring
-                   , criterion >= 1.3.0.0
-                   , cardano-sl-binary
-                   , cardano-sl-block
-                   , cardano-sl-crypto
-                   , cardano-sl-core
-                   , cardano-sl-txp
-                   , cardano-sl-update
-                   , cardano-sl-ssc
-                   , cardano-sl-delegation
-                   , formatting
-                   , universum
-                   , deepseq
+--Benchmark bench-block
+--    type: exitcode-stdio-1.0
+--    main-is: Block.hs
+--    hs-source-dirs: bench
+--    ghc-options: -O2
+--    build-depends:   base
+--                   , bytestring
+--                   , criterion >= 1.3.0.0
+--                   , cardano-sl-binary
+--                   , cardano-sl-block
+--                   , cardano-sl-crypto
+--                   , cardano-sl-core
+--                   , cardano-sl-txp
+--                   , cardano-sl-update
+--                   , cardano-sl-ssc
+--                   , cardano-sl-delegation
+--                   , formatting
+--                   , universum
+--                   , deepseq
```

Comment:
Why comment this out? Need me to fix something here?

Reply:
Yes, in your PR. I'll drop this patch when the base branch will work.

Reply:
BTW you can review by commits ;)